webapp/app.py

```diff
@@ -198,7 +198,16 @@ def bad_request_error(error):
 
 @app.errorhandler(SecurityAPIError)
 def security_api_error(error):
-    return flask.render_template("security-error.html"), 500
+
+    message = error.response.json().get("message")
+
+    if error.response.status_code == 404:
+        return flask.render_template("404.html", message=message), 404
```

Comment:
Did you prefer this syntax to my suggestion of `flask.abort()`? Or did `abort` not work?